synapse/lib/oauth.py

```diff
@@ -73,6 +34,18 @@ def normOAuthTokenData(issued_at, data):
         'refresh_token': data.get('refresh_token'),
     }
 
+az_tfile_envar = 'AZURE_FEDERATED_TOKEN_FILE'
+def _getAzureTokenFile():
+    fp = os.getenv(az_tfile_envar, None)
+    if not fp:
+        return False, {'error': f'{az_tfile_envar} environment variable is not set.'}
```

Comment:
Should this be `mesg` rather than `error` for consistency with other retn use cases?

Reply:
Trying to keep this consistent with other code in the oauth.py file. Will revisit this.

Reply:
error is already used consistently for reporting a user-facing error and identifying when a workflow has failed for some reason.